### PR TITLE
doc: release-notes-latest: Add changes in drivers from last upmerge

### PR DESCRIPTION
**Pull request by anangl**
Only those relevant to NCS were selected.

Signed-off-by: Andrzej Głąbek <andrzej.glabek@nordicsemi.no>

### CODE REVIEW

**Review by ioannisg — APPROVED** [review]

File: doc/nrf/releases/release-notes-latest.rst [hunk 1]
```diff
@@ -155,3 +155,57 @@ The following list summarizes the most important changes inherited from upstream
...
+      * ``LINE_CTRL_RTS``
+      * ``LINE_CTRL_DTR``
+      * ``LINE_CTRL_DCD``
+      * ``LINE_CTRL_DSR``
+
+  * USB:
+
+    * Fixed handling of zero-length packets (ZLP) in the Nordic Semiconductor USB Device Controller driver (usb_dc_nrfx).
+    * Fixed initialization of the workqueue in the usb_dc_nrfx driver, to prevent fatal errors when the driver is reattached.
+    * Fixed handling of the SUSEND and RESUME events in the Bluetooth classes.
```
Comment (jfischer-no) [inline doc/nrf/releases/release-notes-latest.rst:210]: SUSPEND

File: doc/nrf/releases/release-notes-latest.rst [hunk 2]
```diff
@@ -155,3 +155,57 @@ The following list summarizes the most important changes inherited from upstream
...
+    * Added support for the MCUBoot's single application slot mode used with the USB DFU class.
```
Comment (jfischer-no) [inline doc/nrf/releases/release-notes-latest.rst:211]: Not related to drivers, you can remove this line, see https://github.com/nrfconnect/sdk-nrf/pull/3465/commits/8ec92de322ea1957a29763653660b9fa88cc4cf3#diff-786d6262f534cd175913132350738c7670e9561bf4a71816cfa6f76b38c69dccR143

Comment (anangl) [reply doc/nrf/releases/release-notes-latest.rst]: Removed.

**Review by jfischer-no — APPROVED** [review]

File: doc/nrf/releases/release-notes-latest.rst [hunk 3]
```diff
@@ -155,3 +155,56 @@ The following list summarizes the most important changes inherited from upstream
     * ``SYS_CLOCK_HW_CYCLES_TO_NS()``
 
   * Updated :c:func:`k_timer_user_data_get` to take a ``const struct k_timer *timer`` instead of a non-\ ``const`` pointer.
+
+* Drivers:
+
+  * ``DEVICE_INIT()`` macro has been deprecated. Use ``DEVICE_DEFINE()`` instead.
```
Comment (ru-fu) [inline doc/nrf/releases/release-notes-latest.rst:161]:
```suggestion
  * Deprecated the ``DEVICE_INIT()`` macro.
     Use :c:macro:`DEVICE_DEFINE` instead.
```

File: doc/nrf/releases/release-notes-latest.rst [hunk 4]
```diff
@@ -155,3 +155,56 @@ The following list summarizes the most important changes inherited from upstream
...
+
+  * ADC:
+
+    * Improved the default routine that provides sampling intervals, to allow intervals shorter than 1 millisecond.
+
+  * Display:
+
+    * Added support for the ILI9488 display.
+    * Refactored the ILI9340 driver to support multiple instances, rotation, and pixel format changing at runtime.
+      Configuration of the driver instances is now done in Devicetree.
```
Comment (ru-fu) [inline doc/nrf/releases/release-notes-latest.rst:171]:
```suggestion
      Configuration of the driver instances is now done in devicetree.
```

File: doc/nrf/releases/release-notes-latest.rst [hunk 5]
```diff
@@ -155,3 +155,56 @@ The following list summarizes the most important changes inherited from upstream
...
+    * Enhanced the SSD1306 driver to support communication via both SPI and I2C.
+
+  * Flash:
+
+    * Modified the nRF QSPI NOR driver so that it supports also nRF53 Series SoCs.
```
Comment (ru-fu) [inline doc/nrf/releases/release-notes-latest.rst:176]: This one is already added in https://github.com/nrfconnect/sdk-nrf/pull/3465 - can you coordinate with @nvlsianpu ?

**Review by ru-fu — COMMENTED** [review]
Looks good, just some small fixes and a duplicate.

File: doc/nrf/releases/release-notes-latest.rst (hunk 5 above)
Comment (anangl) [reply doc/nrf/releases/release-notes-latest.rst:177]: I guess it should be removed from there and stay here. Is that okay with you @nvlsianpu?

**Review by ru-fu — APPROVED** [review]
Approved, but need to make sure to not include the flash entry twice.

**Pull request merged**